tests/testlib/futures.nim

```diff
@@ -0,0 +1,13 @@
+import
+#   std/sequtils,
+#   std/options,
+#   stew/shims/net as stewNet,
+#   testutils/unittests,
+  chronicles,
+  chronos
+#   libp2p/crypto/crypto
```

Comment:
Here I would delete the lines with unused modules instead of commenting them

Reply:
Ah, yes. Oversight when testing. Thanks!